games-action/openclonk-7.0

```diff
@@ -0,0 +1,118 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you\
+control clonks, small, but witty and nimble humanoid beings. The game\
+is mainly about mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
```

Comment:
It won't build for x86_gcc, so put 8 in front "!x86_gcc2"
Add: SECONDARY_ARCHITECTURES="x86" after this line (that should make building with gcc5 possible)
build the recipe with "hp openclonk_x86" for that